src/Thoth.Json/Encode.fs

```diff
@@ -338,7 +338,17 @@ module Encode =
         Option.map encoder >> Option.defaultWith (fun _ -> nil)
 
     type Auto =
-        static member toString(space : int, value : obj, ?forceCamelCase : bool) : string =
+        static member toString(space : int, value : obj, ?forceCamelCase : bool, ?ignoreMembers: string seq) : string =
```

Comment:
What do you think of renaming `ignoreMembers` to `ignoreProperties`?

For me, members are more when working with class and not record. But I can be wrong.
Also, I believe, `property` is the standard name for JSON object.